locales/ku_KU/duckduckgo.po

```diff
@@ -0,0 +1,5938 @@
+msgstr "Make sure your search is spelled correctly."
+
+#. The name of the Malagasy language
+msgctxt "language_name"
+msgid "Malagasy"
+msgstr "Malagasy"
+
+#. The name of the Malay language
+msgctxt "language_name"
+msgid "Malay"
+msgstr "Malay"
+
+#. The name of the Malayalam language
+msgctxt "language_name"
+msgid "Malayalam"
+msgstr "Malayalam"
+
+msgid "Malaysia"
+msgstr "Malaysia"
+
+msgid "Malaysia (en)"
+msgstr "Malaysia (en)"
+
+#. The name of the Maltese language
+msgctxt "language_name"
+msgid "Maltese"
+msgstr "Maltese"
+
+#. The name of the Maori language
+msgctxt "language_name"
+msgid "Maori"
+msgstr "Maori"
+
+#. Used in 0-click box for local results, e.g. <a href="http://duckduckgo.com/?q=black+lab+bistro">http://duckduckgo.com/?q=black+lab+bistro</a>
+msgid "Map"
+msgstr "Map"
+
+msgctxt "settings"
+msgid "Map Rendering"
+msgstr "Map Rendering"
+
+msgid "Maps"
+msgstr "Nexşe"
+
+msgctxt "feedback form"
+msgid "Maps"
+msgstr "Nexşe"
+
+msgctxt "feedback form"
+msgid "Maps / location info"
+msgstr "Maps / location info"
+
+#. The name of the Marathi language
+msgctxt "language_name"
+msgid "Marathi"
+msgstr "Marathi"
+
+#. Indicates this is a list of matchups (games) between two teams
+msgctxt "Sports module"
+msgid "Matchups"
+msgstr "Matchups"
+
+msgctxt "precise_user_location"
+msgid "Maybe later"
+msgstr "Maybe later"
+
+#. In 0-click box -- for disambiguation.
+msgid "Meanings"
+msgstr "Meanings"
+
+msgctxt "settingsvalue"
+msgid "Medium"
+msgstr "Medium"
+
+#. http://i.imgur.com/Gy9D9Bx.png
+msgctxt "size"
+msgid "Medium"
+msgstr "Medium"
+
+msgctxt "video-duration"
+msgid "Medium"
+msgstr "Medium"
+
+#. Header for front page menu
+msgid "Menu"
+msgstr "Menu"
+
+msgctxt "maps_places"
+msgid "Menu"
+msgstr "Menu"
+
+msgctxt "settingsvalue"
+msgid "Metric (Kilograms, Meters, Celsius)"
+msgstr "Metric (Kilograms, Meters, Celsius)"
+
+msgctxt "settingsvalue"
+msgid "Metric (kilograms, meters, Celsius)"
+msgstr "Metric (kilograms, meters, Celsius)"
+
+msgid "Mexico"
+msgstr "Mexico"
+
+#. Discontinued, don't translate
+msgctxt "placement"
+msgid "Middle"
+msgstr "Middle"
+
+#. The market capitalization of the currently selected stock
+msgctxt "Stocks module"
+msgid "Mkt Cap"
+msgstr "Mkt Cap"
+
+msgctxt "settings"
+msgid "Mobile Instructions (not displayed on settings page)"
+msgstr "Mobile Instructions (not displayed on settings page)"
+
+msgctxt "safe search"
+msgid "Moderate"
+msgstr "Moderate"
+
+msgctxt "settingsvalue"
+msgid "Moderate"
+msgstr "Moderate"
+
+#. Used to point to additional social networking profiles (in results).
+msgid "More"
+msgstr "More"
+
+#. Link to load more page context
+msgctxt "additional"
+msgid "More"
+msgstr "Bêtir"
+
+#. As in Learn More
+msgctxt "frontpage"
+msgid "More"
+msgstr "Bêtir"
+
+msgid "More Images"
+msgstr "More Images"
+
+msgid "More Info"
+msgstr "Bêtir Zanyarî"
+
+msgctxt "Ads GDPR, internal use only. Do not change"
+msgid "More Info"
+msgstr "Bêtir Zanyarî"
```

Comment:
Please keep this string untranslated. It's for internal use only. Thanks! 🙂
```suggestion
msgstr ""
```

Reply:
Fixed!